core/io/dir_access.cpp

```diff
@@ -460,28 +460,30 @@ Error DirAccess::_copy_dir(Ref<DirAccess> &p_target_da, String p_to, int p_chmod
 	return OK;
 }
 
-Error DirAccess::copy_dir(String p_from, String p_to, int p_chmod_flags, bool p_copy_links) {
+Error DirAccess::copy_dir(const String &p_from, const String &p_to, int p_chmod_flags, bool p_copy_links) {
```

Comment:
For now I'd remove any case that requires copying in the body, let's keep this to just const changes, and possibly changing the iterators if needed

Reply:
Got it!